tests/TestLDFFormat.cpp

```diff
@@ -22,7 +22,11 @@ int TestLDFFormat(int argc, char* *const argv) {
 	ASSERT_EQ(((LDFData<std::u16string>* )data)->GetValue(), u"VALUE");
 
 	// Check that the serialization is correct
-	ASSERT_EQ(data->GetString(), "KEY=0:VALUE");
+	if (data->GetString() != "KEY=0:VALUE"){
```

Comment:
why did you change this?

Reply:
I was just temporarily trying to work out why it wasn't working on Linux, the test was working on Windows but wasn't on Linux